reco_utils/azureml/aks_utils.py

```diff
@@ -0,0 +1,57 @@
+from math import ceil, floor
+
+def qps_to_replicas(target_qps, processing_time, max_qp_replica=1, target_utilization=0.7, verbose=True):
+    """

```

Comment:
A minor comment. In the other code we are using the docstring format like this:
```
"""Provide a rough estimate ...
```
Also, another annoying thing is that if we don't add a new line between the end of `Args` and `Returns`, when we use sphinx, it won't parse the docstrings. I learned this the hard way because I have to add a new line in all the code of a library that I had, and it was very time consuming.